web-app/js/portal/filter/BaseFilterPanel.js

```diff
@@ -125,8 +125,8 @@ Portal.filter.BaseFilterPanel.newFilterPanelFor = function(cfg) {
         newFilterPanel = new Portal.filter.GeometryFilterPanel(cfg);
     }
     else {
-        //Filter hasn't been defined
-        log.error("Could not create filter panel for type'" + type + "'");
+        log.error("Unhandled filter type '" + type + "' for layer '" + cfg.layer.wmsName + "'");
```

Comment:
So when testing, I realized it'd be beneficial to have also the filter name here, but it's not a biggie because we have already the layer name. Up to you, not blocking me from merging this.